lib/catalog/owid/catalog/processing_log.py

```diff
@@ -68,6 +68,9 @@ class ProcessingLog(List[LogEntry]):
     # hack for dataclasses_json
     __args__ = (LogEntry,)
 
+    def __init__(self, entries: List[LogEntry | Dict[str, Any]] = []):
+        super().__init__([entry if isinstance(entry, LogEntry) else LogEntry.from_dict(entry) for entry in entries])
```

Comment:
This is a bit obscure to me, but I take your word on this :-D

Reply:
It accepts both `LogEntry` and `dict` when initializing. Clarified that in a comment.